dashboard/app/models/pd/application/active_application_models.rb

```diff
@@ -4,6 +4,7 @@ module ActiveApplicationModels
       include ApplicationConstants
 
       APPLICATION_CURRENT_YEAR = YEAR_19_20
+      DASHBOARD_APPLICATION_YEAR = YEAR_19_20
```

Comment:
From reading the code above, this variable actually means just "teacher application", not both teacher and facilitator application. Maybe adding a comment here to avoid confusion in the future.

Reply:
I wonder why we show current year teacher application but show all facilitator applications? The facilitator application also changes year over year.